pkg/apis/workspaces/v1alpha2/devworkspaceTemplateSpec.go

```diff
@@ -12,6 +12,16 @@ type DevWorkspaceTemplateSpec struct {
 
 // +devfile:overrides:generate
 type DevWorkspaceTemplateSpecContent struct {
+	// Map of string attributes.
+	// Attribute values can be referenced throughout the devfile in string type fields in the form {{attribute-key}}
+	// except for schemaVersion, metadata, parent source. Exception to the string field also include element's key identifiers
+	// (command id, component name, endpoint name, project name, etc.) and their references(events, command's component, container's
+	// volume mount name, etc.) and string enums(command group kind, endpoint exposure, etc.)
+	// +optional
+	// +patchStrategy=merge
+	// +devfile:overrides:include:omitInPlugin=true,description=Overrides of attributes encapsulated in a parent devfile.
+	Attributes map[string]string `json:"attributes,omitempty" patchStrategy:"merge"`
```

Comment:
Moving to `map[string]string` complicates the issue further IMO. Now we have adjacent attributes fields that take different types of data.

Reply:
@amisevsk so do you recommend a rename of this field or stay as Attributes and move back to free-form yaml? Btw I'm not sure if you have read my prev comment here https://github.com/devfile/api/pull/352#discussion_r589697319, but we're definitely interested in keeping metadata.Attributes.

If we want to keep it as `map[string]string`, then it might be a good idea to rename it? 🤔